App.tsx

```diff
@@ -1,19 +1,25 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
I see what you did here. Please try to avoid this at any costs 👀

Reply:
Understood fixed !